src/Cms/Plugin.php

```diff
@@ -173,6 +184,18 @@ public function option(string $key)
         return $this->kirby()->option($this->prefix() . '.' . $key);
     }
 
+    /**
+     * Kirby plugin register
+     *
+     * @param string $name
+     * @param array $extends
+     * @return \Kirby\Cms\Plugin
+     */
+    public static function register(string $name, array $extends = [])
+    {
+        return App::instance()->plugin($name, $extends);
```

Comment:
If these are static methods, they should also only call static methods (and not on the app instance)

Reply:
You are creating new static methods, e.g. `Plugin::register()` but right now those create a instance/object of Kirby via `App::instance()` - I think this shouldn't be the case, but rather also call `App::plugin()` statically.

Reply:
Done 👍